src/thread/stack.cr

```diff
@@ -0,0 +1,16 @@
+# :nodoc:
+class Thread
+  # :nodoc:
+  struct Stack
+    property top : Pointer(Void)  # Lowest address of a writable byte
```

Comment:
As a general rule always put the comments one line above the code in question, otherwise they'll be misassigned by the doc generator (even though that's not the case here atm, since it's nodoc-ed).

Reply:
These comments aren't written in a style that's suitable for public docs either, so I'm going to leave the formatting as-is.  `crystal tool format` was happy to line up the comments to start in the same column for me when I had left them ragged, so I imagine this commenting style is not entirely foreign.